app/views/item/packages/index/ShippingMarkView.scala.html

```diff
@@ -0,0 +1,56 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.item.packages.index.routes
+@import models.LocalReferenceNumber
+@import forms.Constants.maxShippingMarkLength
+@import viewmodels.components.InputCharacterCountViewModel._
+@import views.html.components.{Button, ErrorSummary, InputCharacterCount}
+
+
+@this(
+    mainTemplate: MainTemplate,
+    formHelper: FormWithCSRF,
+    inputCharacterCount: InputCharacterCount,
+    errorSummary: ErrorSummary,
+    button: Button
+)
+
+@(form: Form[String], lrn: LocalReferenceNumber, mode: Mode, itemIndex: Index, packageIndex: Index)(implicit request: Request[_], messages: Messages)
+
+@mainTemplate(
+    title = messages("item.packages.index.shippingMark.title"),
+    showBackLink = true,
+    lrn = lrn
+) {
+
+    @formHelper(action = routes.ShippingMarkController.onSubmit(lrn, mode, itemIndex, packageIndex), Symbol("autoComplete") -> "off") {
+
+        @errorSummary(form.errors)
+
+        @inputCharacterCount(
+            field = form("value"),
+            inputCharacterCountType = OrdinaryInputCharacterCount(
+                heading = messages("item.packages.index.shippingMark.heading"),
+                caption = Some(messages("item.packages.caption", itemIndex.display)),
+            ),
+            label = messages("item.packages.index.shippingMark.heading"),
```

Comment:
This has the heading in here, is that correct?

Reply:
Yep I believe so, all text pages hold a label that references the heading. Let's verify this in the UI before approving though.